src/main/java/io/seqera/tower/cli/commands/LaunchesCmd.java

```diff
@@ -49,7 +54,7 @@
     @Option(names = {"-c", "--compute-env"}, description = "Compute environment name (defaults to workspace primary)")
     String computeEnv;
 
-    @Option(names = {"-w", "--work-dir"}, description = "Working directory")
+    @Option(names = {"--work-dir"}, description = "Working directory")
```

Comment:
mmmh: I don't like that the short version of the argument has been removed. Can we use -d because it is not consistent having all the other with the short and long version while this one only has the long option.

Reply:
Some params have no short version:

https://github.com/seqeralabs/tower-cli/blob/510a77f39e8a330bf9f2cb353f0e51044d19382f/src/main/java/io/seqera/tower/cli/commands/computeenvs/platforms/AwsBatchManualPlatform.java#L25-L30

In addition `-d` has conflicts with the `description` param, but `-k` is available.

Reply:
I really don't like to do not use `-w` as a shortcut for `--work-dir`. The main reason is because we are trying to use as much as possible the same options that is using `nextflow run` command line. 

I was thinking that if in a future we implement the "profile" feature so that you can switch to permanently use a different workspace, most likely the `--workspace` option won't be used so much. My proposal would be to change the workspace option to use this `"-ws", "--workspace"` or maybe this `"--ws", "--workspace"`. And then keep using `-w` for `workDir`. Opinions?

Reply:
`-w`, `--work-dir `

and 

`-W`, `--workspace` 

?

Reply:
Yes, nice. I didn't thought about this option. I like it.

Reply:
tho, I think it's much more needed to being able to specify the workspace, more than the work dir. Would not make sense to only have `--work-dir` and use `-w` for the workspace?

Reply:
Currently you can also specify the workspace using the `TOWER_WORKSPACE_ID` environment variable. And in a future I imagine something similar to AWS CLI that you can define different "profiles" and switch between them so you can define the new active profile. And then you don't need to pass the workspace on all the commands while you are working on the same workspace, just once to change the active workspace. If we've this, then I think that to pass the workspace using command line options won't be used so much. This is the main reason why I think that it's a petty to do not use `-w` for work directory as in `nextflow run`.

Reply:
I imagine a command similar to this to switch context:
```
tw set-context --current --workspace=<...> --token=<...> --url=<...>
```

Reply:
> This is the main reason why I think that it's petty to do not use `-w` for the work directory as in `nextflow run`.

Still, we may wish to keep the use of `-w` to define a workspace on the fly despite the profile feature. Also, for backward compatibility.

Personally, I would stick with `--work-dir `(and perhaps `-k`, since it is not compromised) and keep `-w` and `-o `for workspace and organization.

Keep in mind we do not need to have a single word param shortcut for all options, we wouldn't have enough letters.